setup.cfg

```diff
@@ -1,3 +1,41 @@
 [wheel]
-universal=1
+universal = 1
+
+[metadata]
+name = widgetastic.core
+author = Milan Falesnik
+author_email = mfalesni@redhat.com
+description = Making testing of UIs fantastic
+long_description= file: README.rst
+license = Apache license
+url = https://github.com/RedHatQE/widgetastic.core
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
```

Comment:
@mshriver as we are supporting `py3.5`. can we include in `classifiers` and `tox` (It will skip if not found).

Reply:
yes! sorry I removed it, then realized that pypy latest support was 3.5 and re-added. Thanks!